plugins/openshift/skills/generating-ovn-topology/scripts/collect-ovn-data.py

```
@@ -0,0 +1,610 @@
+        Also populates self.ovnkube_node_pods with discovered ovnkube-node pods.
+
+        Returns:
+            True if successful, False otherwise.
+        """
+        print("=" * 50)
+        print("COLLECTING POD INFORMATION")
+        print("=" * 50)
+
+        try:
+            # Try JSON parsing first (more reliable)
+            result = subprocess.run(
```

Ruff (0.14.3)

`subprocess` call: check for execution of untrusted input

(S603)